providers/src/airflow/providers/microsoft/azure/hooks/adx.py

```diff
@@ -206,11 +206,11 @@ def get_required_param(name: str) -> str:
 
         return KustoClient(kcsb)
 
-    def run_query(self, query: str, database: str, options: dict | None = None) -> KustoResponseDataSetV2:
+    def run_query(self, query: str, database: str, options: dict | None = None) -> KustoResponseDataSet:
         """
         Run KQL query using provided configuration, and return KustoResponseDataSet instance.
 
-        See: `azure.kusto.data.response.KustoResponseDataSet`
+        See: ``azure.kusto.data.response.KustoResponseDataSet``
```

Comment:
Reverting my miserable attempt :-(
```suggestion
        See: `azure.kusto.data.response.KustoResponseDataSet`
```